lib/client.js

```diff
@@ -229,10 +243,28 @@ Client.prototype = {
   /// client.trigger('activation') // activating!
   /// ```
   trigger: function(name, data) {
-    if (!this._messageHandlers[name]) { return false; }
-    this._messageHandlers[name].forEach(function(handler) {
-      handler(data);
-    });
+    if (!this._messageHandlers[name] || repliesPending[name]) { return false; }
```

Comment:
> Could we put them both on the client object?

Yes, we can do that.  If we're doing so, we should probably put the `pendingPromises` module-local var into the client to.  This would have the advantage of making it easier to allow multiple clients in the one app, (when/if we want to do that).